invokeai/app/api/routers/models.py

```diff
@@ -29,20 +27,34 @@ class CkptModelInfo(ModelInfo):
     width: Optional[int] = Field(description="The width of the model")
     height: Optional[int] = Field(description="The height of the model")
 
-
 class DiffusersModelInfo(ModelInfo):
     format: Literal['diffusers'] = 'diffusers'
 
     vae: Optional[VaeRepo] = Field(description="The VAE repo to use for this model")
     repo_id: Optional[str] = Field(description="The repo ID to use for this model")
     path: Optional[str] = Field(description="The path to the model")
 
+class CreateModelRequest (BaseModel):
```

Comment:
Remove space after class name and before opening parenthesis (on all 4 of these).

Reply:
👍